pkg/scheduler/framework/types.go

```diff
@@ -91,13 +91,15 @@ type ClusterEventWithHint struct {
 // QueueingHintFn returns a hint that signals whether the event can make a Pod,
 // which was rejected by this plugin in the past scheduling cycle, schedulable or not.
 // It's called before a Pod gets moved from unschedulableQ to backoffQ or activeQ.
+// If it returns an error, the scheduling queue behaves like QueueingHintFn returned QueueAfterBackoff so that
```

Comment:
Let's make it more precise, like
```suggestion
// If it returns an error, we'll take the returned QueueingHint as `QueueAfterBackoff` at the caller whatever we returned here so that
```